src/Workspaces/Core/Portable/ExternalAccess/UnitTesting/Api/UnitTestingImmutableArrayExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.UnitTesting.Api
+{
+    internal static class UnitTestingImmutableArrayExtensions
+    {
+        public static ImmutableArray<byte> UnitTesting_ToImmutable(this MemoryStream stream)
```

Comment:
💡 This doesn't need the `UnitTesting_` prefix. The same applies for other members of this class.

```suggestion
        public static ImmutableArray<byte> ToImmutable(this MemoryStream stream)
```